src/Analysis/GWASApp/Components/Covariates/CustomDichotomousCovariates.jsx

```diff
@@ -21,6 +21,14 @@ const CustomDichotomousCovariates = ({
   const handleDichotomousSubmit = () => {
     const dichotomous = {
       variable_type: 'custom_dichotomous',
+      cohort_names: [
+        firstPopulation.cohort_name,
+        secondPopulation.cohort_name,
+      ],
+      cohort_sizes: [
+        firstPopulation.size,
+        secondPopulation.size,
+      ],
```

Comment:
why this change?

Reply:
The Euler diagram requires sizes and labels (key value pairs with key names .cohort_name and .size) in order to render the visualization. Before the attrition table didn't need to render a Euler diagram so this information wasn't stored, only the  cohort_ids were stored. To minimize refactoring, the additional information is now stored in arrays the same way the cohort_ids are stored.